src/services/rfqm_service.ts

```diff
@@ -108,6 +127,7 @@ export interface SubmissionsMap {
 export type FetchFirmQuoteResponse = MetaTransactionRfqmQuoteResponse;
 export type SubmitRfqmSignedQuoteParams = MetaTransactionSubmitRfqmSignedQuoteParams;
 export type SubmitRfqmSignedQuoteResponse = MetaTransactionSubmitRfqmSignedQuoteResponse;
+export type StatusResponse = OrderStatusPending | OrderStatusSubmitted | OrderStatusFailed | OrderStatusSuccessful;
```

Comment:
while I like this in Typescript, I'm less convinced it would be convenient without the type bindings. My preference would simply to use a `transactions` array. For each `status` - would just do
- when `pending`,  `transactions = []`
- when `successful`,  `transactions = [txn]`
- when `failed` | `submitted`, `transactions = txns`

Reply:
grpc wen

Reply:
done in the david/status PR